### DISCUSSION TITLE
Install Coolify On Github Organization

### DISCUSSION THREAD

**Discussion by lewismunene020** [coollabsio/coolify#2649] [Feature Requests]
This  does'nt  currently work  i  have to  setup deploy  keys  every now and then  which is   tiresome

**Comment by nmpereira** [comment]
I only started using coolify a few days ago, but this seems to be supported. I have my repo private on a GitHub org and I can use a GitHub app integration to automatically trigger deploys.

**Comment by AndrewPetz** [comment]
I'm having the same issue. I'm the sole member and owner of the Organization.

![image](https://github.com/user-attachments/assets/113fc15b-9826-447e-ab29-53f979a2dc22)

Clicking "Register Now" sends me to this page:

![image](https://github.com/user-attachments/assets/0357a439-eeb1-4875-87fe-6d05a0ef1f95)

As far as I can tell there are no restrictions on my organization related to Apps, so I'm not sure what to do here.

**Comment by lewismunene020** [comment]
Check theis image and create a  github app  for  your organization using  your oganization slug name

![image](https://github.com/user-attachments/assets/0aad2aa0-dfcd-42a5-8c04-1a51f5149f24)

**Comment by lewismunene020** [comment]
This issue was  resolved  .
It is  currently working
